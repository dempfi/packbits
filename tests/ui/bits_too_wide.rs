use packbits as _;

#[packbits::pack(bytes = 1)]
struct Bad {
    #[bits(33)]
    a: u32,
}

fn main() {}
