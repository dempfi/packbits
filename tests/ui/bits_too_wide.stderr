error: bits width must be 1..=32
 --> tests/ui/bits_too_wide.rs:5:5
  |
5 |     #[bits(33)]
  |     ^
